crates/re_space_view_spatial/src/scene/mod.rs

```diff
@@ -126,3 +126,17 @@ pub fn load_keypoint_connections(
         }
     }
 }
+
+/// Returns the view coordinates used for 2D (image) views.
+///
+/// TODO(#1387): Image coordinate space should be configurable.
+pub fn image_view_coordinates() -> re_components::ViewCoordinates {
+    // Typical image spaces have
+    // - x pointing right
+    // - y pointing down
+    // - z pointing into the image plane (this is convenient for reading out a depth image which has typically positive z values)
+    re_components::ViewCoordinates::from_up_and_handedness(
+        re_components::coordinates::SignedAxis3::NEGATIVE_Y,
+        re_components::coordinates::Handedness::Right,
+    )
```

Comment:
`from_up_and_handedness` is free to return any coordinate system that fits the input constraint (there are several!).

Better to be explicit:

```suggestion
    re_components::ViewCoordinates::rdf()
```